samples/tfm_integration/tfm_psa_arch_tests/README.rst

```diff
@@ -0,0 +1,127 @@
+.. _tfm_psa_arch_api:
+
+TF-M psa-arch-tests
+###################
+
+Overview
+********
+This is a sample for Zephyr PSA level 1 with TF-M integrated.
+
+Zephyr uses Trusted Firmware (TF-M) Platform Security Architecture (PSA) APIs
+to run in a secure configuration with Zephyr itself in a non-secure configuration.
+The sample prints test info to the console as a multi-thread application.
+
+The psa-arch-tests provides differents test suits, this example supports
+crypto, protected_storage, initial_attestation. Just config the suite by
+prj.conf:
+
+.. code-block:: bash
+
+   CONFIG_API_TEST_SUIT="crypto"
+
+Building and Running
+********************
+
+This project outputs test status and info to the console. It can be built and
+executed on MPS2+ AN521.
+
+On MPS2+ AN521:
+===============
+
+1. Build Zephyr with a non-secure configuration (``-DBOARD=mps2_an521_nonsecure``).
+
+.. code-block:: bash
+
+   cd $ZEPHYR_ROOT/samples/tfm_integration/tfm_psa_arch_tests/
+   mkdir build
+   cd build
+   cmake -GNinja -DBOARD=mps2_an521_nonsecure ..
+   ninja -v
+
+2. Copy application binary files (mcuboot.bin and tfm_sign.bin) to ``<MPS2 device name>/SOFTWARE/``.
+3. Open ``<MPS2 device name>/MB/HBI0263C/AN521/images.txt``.
+4. Update the ``AN521/images.txt`` file as follows:
+
+.. code-block:: bash
+
+   TITLE: Versatile Express Images Configuration File
+
+   [IMAGES]
+   TOTALIMAGES: 2 ;Number of Images (Max: 32)
+
+   IMAGE0ADDRESS: 0x10000000
+   IMAGE0FILE: \SOFTWARE\mcuboot.bin  ; BL2 bootloader
+
+   IMAGE1ADDRESS: 0x10080000
+   IMAGE1FILE: \SOFTWARE\tfm_sign.bin ; TF-M with application binary blob
+
+5. Reset MPS2+ board.
+
+
+On V2M Musca B1:
+================
+
+1. Build Zephyr with a non-secure configuration (``-DBOARD=v2m_musca_b1_nonsecure``).
+
+.. code-block:: bash
+
+   cd $ZEPHYR_ROOT/samples/tfm_integration/tfm_psa_arch_tests/
+   mkdir build
+   cd build
+   cmake -GNinja -DBOARD=v2m_musca_b1_nonsecure ..
+   ninja -v
+
+2. The binaries will be sign and combine automatically into build folder (tfm_zephyr.hex).
+3. Plunge in Musca B1 board, and press power on button.
+4. Drag and drop tfm_zephyr.hex to root of the MUSCA_B(E:) drive.
```

Comment:
Hmmm.  Sounds like you did this on a Windows system.  Can you write this to work on both windows and Linux?